### DISCUSSION TITLE
fetch_user_post 接口无法使用

### DISCUSSION THREAD

**Discussion by arthur-dy-lee** [Evil0ctal/Douyin_TikTok_Download_API#397] [Q&A]
1. tiktok 接口fetch_user_post无法使用， Page cursor=0的时候，多数是可以使用，当 =1 或 =2其它时候就无法返回值了，虽然code=200，但没有内容。https://douyin.wtf/docs也不行。Douyin的这个fetch_user_post接口，也有相同的问题，你提供网站上的demo也无法使用
2. 有没有群里，电报，微信，discord可以进群交流？

**Comment by arthur-dy-lee** [comment]
问题解决了， cursor使用的不是 0,1,2 ，而是时间戳

**Comment by arthur-dy-lee** [comment]
有没有群里，电报，微信，discord可以进群交流？

**Reply by Evil0ctal** [reply]
我们有微信群，但是人数超过200人了，无法通过二维码加入，请加我的微信拉你加群：Evil0ctal
